packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -458,6 +472,18 @@ class _ZoomExitTransitionState extends State<_ZoomExitTransition> with _ZoomTran
     super.didUpdateWidget(oldWidget);
   }
 
+  @override
```

Comment:
Instead of duplicating this, could it have been implemented in the _ZoomTransitionBase if that mixin were to be made a mixin on State?